python/tests/plugins/lsp_server/providers/test_hover_provider.py

```diff
@@ -32,10 +32,10 @@ async def hover(self, file_name: str, line: int = 0, character: int = 0) -> Hove
         )
 
     async def test_handles_hover_request(self):
-        res: HoverResponse = await self.hover(TEST_DOCUMENT_NAME)
-        self.assertEqual(res.get_content(), self.active_context.get_definition_by_name("schema").content)
+        res: HoverResponse = await self.hover(TEST_DOCUMENT_NAME, 1, 8)
+        self.assertEqual(res.get_content(), self.active_context.get_definition_by_name("DataA").content)
 
     async def test_no_hover_when_nothing_under_cursor(self):
-        await self.write_document(TEST_DOCUMENT_NAME, f"\n{TEST_DOCUMENT_CONTENT}")
+        await self.write_document(TEST_DOCUMENT_NAME, f" \n{TEST_DOCUMENT_CONTENT}")
```

Comment:
What exactly was the reasoning for changing this?

Reply:
The changes with symbol didn't hanlde it -- I fixed the internal logic to account for empty lines.